src/json/state_machine_input.json

```diff
@@ -119,6 +119,27 @@
       ]
     }
   },
+  "vpcpreconfig":{  
```

Comment:
What is the difference between manageVpcPreConfig and vpcpreconfig?

Reply:
vpcpreconfig is like parent JSON element and manageVpcPreConfig is the child JSON element.